### PR TITLE
DACT-958: Skeleton director details update page

### PR DESCRIPTION
**Pull request by olaych**
https://companieshouse.atlassian.net/browse/DACT-958

### CODE REVIEW

File: src/controllers/active.directors.controller.ts [hunk 1]
```diff
@@ -78,13 +79,24 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
   const transactionId = urlUtils.getTransactionIdFromRequestParams(req);
   const appointmentId = req.body.appointmentId;
   const session: Session = req.session as Session;
-  
+  console.log(`req bidy is ${JSON.stringify(req.body["update_director_details"])}`)
```
Comment (cdrainCH) [inline src/controllers/active.directors.controller.ts:82]: Minor - console.log not removed

File: test/controllers/update/update.director.details.controller.unit.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    it("should catch error if error occurred", async () => {
+      mockGetOfficerFiling.mockRejectedValue("Error getting officer filing");
+      const response = await request(app).get(PAGE_URL);
+      expect(response.text).not.toContain(PAGE_HEADING);
+      expect(response.text).toContain(ERROR_PAGE_HEADING)
+    });
+  });
+
+  describe("POST tests", () => {
+    it("Should redirec on submission", async () => {
```
Comment (cdrainCH) [inline test/controllers/update/update.director.details.controller.unit.ts:54]: Minor - redirect is missing the "t" at the end

File: views/includes/corporate-directors.html [hunk 3]
```diff
@@ -20,11 +20,22 @@ <h2 class="govuk-summary-card__title">{{officerCard.officer.name}}</h2>
...
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key"></dt>
+        <dd class="govuk-summary-list__value"></dd>
+      </div>
+    </dl>
       <div class="govuk-!-padding-top-6">
+        {% if updateEnabled === "true" %}
+        <button id="directorUpdateButton" name="update_director_details" class="govuk-button govuk-button--secondary govuk-!-margin-right-2" data-module="govuk-button" value="{{officerCard.appointmentId}}">
+          View and update details
+        </button>
```
[the remark this answers is not in the document]
Comment (olaych) [reply views/includes/corporate-directors.html]: Confirmed with BA and ticket narrative was a mistake. Now removed from corporate directors

File: test/controllers/update/update.director.details.controller.unit.ts (hunk 2 above)
Comment (olaych) [reply test/controllers/update/update.director.details.controller.unit.ts]: fixed

**Review by cdrainCH — APPROVED** [review]

**Review by mdesaikns — DISMISSED** [review]

**Review by cdrainCH — APPROVED** [review]

**Pull request merged**